google/cloud/dialogflowcx_v3/types/test_case.py

```diff
@@ -198,7 +198,7 @@ class UserInput(proto.Message):
         r"""The input from the human user.
 
         Attributes:
-            input_ (google.cloud.dialogflowcx_v3.types.QueryInput):
+            input (google.cloud.dialogflowcx_v3.types.QueryInput):
```

Comment:
Note this as breaking in the docs, but do not take a major version bump since this library is pre-GA.